test/config/integration_tests.yml

```diff
@@ -72,6 +72,14 @@ testCases:
 - name: nested-ss
   configFile: test/config/examples/mnist-nested-search-space.yml
 
+- name: classic-nas-pytorch
```

Comment:
Is this a tf2 test case?  if yes, need to be moved into test/config/integration_tests_tf2.yml

Reply:
this one can be tested with tf 1.x, I will add another one in `test/config/integration_tests_tf2.yml`